apps/frontend/src/libs/hooks/use-pagination/libs/constants/default-values.constant.ts

```diff
@@ -0,0 +1,6 @@
+const DEFAULT_VALUES = {
+	PAGE: 1,
+	PAGE_SIZE: 10,
+};
```

Comment:
we will use FIRST_PAGE as the default page, so we don't need a separate constant for it
```suggestion
const DEFAULT_PAGE_SIZE = 10;
```
Don't forget about file name